api/src/context.rs

```diff
@@ -670,7 +670,9 @@ impl Context {
             .into_iter()
             .map(|t| {
                 // Update the timestamp if the next block occurs
-                if let Some(txn) = t.transaction.try_as_block_metadata() {
+                if let Some(txn) = t.transaction.try_as_block_metadata_ext() {
```

Comment:
nit:
```
if let Some(txn) = t.transaction.try_as_block_metadata_ext().or_else(|| t.transaction.try_as_block_metadata()) {
    timestamp = txn.timestamp_usecs();
}
```

Reply:
not possible since the return types in both cases are diferent.